app/views/layouts/_support_header.html.erb

```diff
@@ -1,3 +1,4 @@
+<button type="button" class="govuk-header__menu-button govuk-js-header-toggle" aria-controls="navigation" aria-label="Show or hide Top Level Navigation">Menu</button>
```

Comment:
nit: the sign-in page for support includes this header, so there's an empty dropdown

Reply:
Will raise a separate ticket.